pkgs/servers/sql/postgresql/ext/postgis.nix

```diff
@@ -43,14 +45,13 @@ stdenv.mkDerivation rec {
     sed -i "s|\$(DESTDIR)\$(PGSQL_BINDIR)|$prefix/bin|g
             " \
         "raster/scripts/python/Makefile";
-  '';
-
-  preInstall = ''
     mkdir -p $out/bin
+    ln -s ${postgresql}/bin/postgres $out/bin/postgres
   '';
 
   # create aliases for all commands adding version information
   postInstall = ''
+    rm $out/bin/postgres
     for prog in $out/bin/*; do # */
```

Comment:
Although this is not a change in this PR, do you know why there's a `*/` comment here?

Reply:
That clearly seems to unbreak syntax highlighting in editors that get confused by `/*` on that same line.  One can see similar ones all over nixpkgs.